packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -109,6 +115,15 @@ class CupertinoNavigationBar extends StatelessWidget implements ObstructingPrefe
   /// behind it.
   final Color backgroundColor;
 
+  /// The bottom border color of the navigation bar.
+  ///
+  /// If a border color is null or transparent, the tab bar will not display a border.
```

Comment:
'tab bar's are very specific things in iOS
https://developer.apple.com/ios/human-interface-guidelines/bars/tab-bars/

Call it 'nav bar' instead